src/applications/gi/reducers/calculator.js

```diff
@@ -51,10 +51,23 @@ export default function (state = INITIAL_STATE, action) {
         convertedValue = +value.replace(/[^0-9.]+/g, '');
       }
 
-      const newState = {
+      let newState = {
         [field]: convertedValue,
       };
 
+      if (field === 'yellowRibbonDegreeLevel') {
```

Comment:
`degreeLevel` filters `divisionProfessionalSchool`

This creates a new list of options for `divisionProfessionalSchool` and sets its input value to its first option. 

This is the main reason the new attribute is stored on the wrong part of the state. I thought about doing this in the selector but it's already way too big and the input values are not selected- their state is just straight copied onto the props. 

![38958450-2411004a-4312-11e8-850f-71fdf5ac91c5-1](https://user-images.githubusercontent.com/4998130/41384213-c394e564-6f31-11e8-9cb4-02f92b38c905.gif)